contracts/lib/StringHelpers.sol

```diff
@@ -58,7 +58,7 @@ library StringHelpers {
     }
 
     /**
-     * Translates a bytes32 into an ascii hexadecimal representation of those bytes
+     * Translates a bytes32 into an ascii hexadecimal representation of those bytes "0x..."
```

Comment:
"0x..." isn't super clear to me. Does this mean the returned string will always start with "0x"?

Reply:
yeah